[3.18.0] using user profile, fail with [Could not initialize class org.apache.cassandra.utils.UUIDGen]

following command
```
cassandra-stress user profile=/jenkins/workspace/scylla-staging/dtest-pytest-gating/scylla-dtest/cqlsh_tests/blogposts.yaml ops(insert=1) n=10000 cl=ALL no-warmup -rate threads=50 -node 127.0.76.1
```

is failing right away as such:
```
 java.lang.NoClassDefFoundError: Could not initialize class org.apache.cassandra.utils.UUIDGen
	at org.apache.cassandra.stress.generate.values.TimeUUIDs.generate(TimeUUIDs.java:49)
	at org.apache.cassandra.stress.generate.values.TimeUUIDs.generate(TimeUUIDs.java:29)
	at org.apache.cassandra.stress.generate.PartitionIterator$MultiRowIterator.fill(PartitionIterator.java:653)
	at org.apache.cassandra.stress.generate.PartitionIterator$MultiRowIterator.fill(PartitionIterator.java:622)
	at org.apache.cassandra.stress.generate.PartitionIterator$MultiRowIterator.reset(PartitionIterator.java:270)
 at org.apache.cassandra.stress.generate.PartitionIterator.reset(PartitionIterator.java:107)
	at org.apache.cassandra.stress.operations.PartitionOperation.reset(PartitionOperation.java:115)
 at org.apache.cassandra.stress.operations.PartitionOperation.ready(PartitionOperation.java:101)
	at org.apache.cassandra.stress.StressAction$StreamOfOperations.nextOp(StressAction.java:370)
	at org.apache.cassandra.stress.StressAction$Consumer.run(StressAction.java:483)
 Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.AssertionError: java.lang.reflect.InaccessibleObjectException: Unable to make field private int java.io.FileDescriptor.fd accessible: module java.base does not "opens java.io" to unnamed module @7b139eab [in thread "Thread-45"]
	at org.apache.cassandra.utils.FBUtilities.getProtectedField(FBUtilities.java:664)
 at org.apache.cassandra.utils.NativeLibrary.<clinit>(NativeLibrary.java:81)
	at org.apache.cassandra.utils.UUIDGen.hash(UUIDGen.java:388)
 at org.apache.cassandra.utils.UUIDGen.makeNode(UUIDGen.java:367)
	at org.apache.cassandra.utils.UUIDGen.makeClockSeqAndNode(UUIDGen.java:300)
	at org.apache.cassandra.utils.UUIDGen.<clinit>(UUIDGen.java:41)
 ... 10 more
 Exception in thread "Thread-36" java.lang.NoClassDefFoundError: Could not initialize class org.apache.cassandra.utils.UUIDGen
 at org.apache.cassandra.stress.generate.values.TimeUUIDs.generate(TimeUUIDs.java:49)
	at org.apache.cassandra.stress.generate.values.TimeUUIDs.generate(TimeUUIDs.java:29)
	at org.apache.cassandra.stress.generate.PartitionIterator$MultiRowIterator.fill(PartitionIterator.java:653)
	at org.apache.cassandra.stress.generate.PartitionIterator$MultiRowIterator.fill(PartitionIterator.java:622)
	at org.apache.cassandra.stress.generate.PartitionIterator$MultiRowIterator.reset(PartitionIterator.java:270)
 at org.apache.cassandra.stress.generate.PartitionIterator.reset(PartitionIterator.java:107)
 at org.apache.cassandra.stress.operations.PartitionOperation.reset(PartitionOperation.java:115)
	at org.apache.cassandra.stress.operations.PartitionOperation.ready(PartitionOperation.java:101)
	at org.apache.cassandra.stress.StressAction$StreamOfOperations.nextOp(StressAction.java:370)
 at org.apache.cassandra.stress.StressAction$Consumer.run(StressAction.java:483)
 Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.AssertionError: java.lang.reflect.InaccessibleObjectException: Unable to make field private int java.io.FileDescriptor.fd accessible: module java.base does not "opens java.io" to unnamed module @7b139eab [in thread "Thread-45"]
	at org.apache.cassandra.utils.FBUtilities.getProtectedField(FBUtilities.java:664)
 at org.apache.cassandra.utils.NativeLibrary.<clinit>(NativeLibrary.java:81)
	at org.apache.cassandra.utils.UUIDGen.hash(UUIDGen.java:388)
 at org.apache.cassandra.utils.UUIDGen.makeNode(UUIDGen.java:367)
	at org.apache.cassandra.utils.UUIDGen.makeClockSeqAndNode(UUIDGen.java:300)
	at org.apache.cassandra.utils.UUIDGen.<clinit>(UUIDGen.java:41)
 ... 10 more
 java.lang.NoClassDefFoundError: Could not initialize class org.apache.cassandra.utils.UUIDGen
 at org.apache.cassandra.stress.generate.values.TimeUUIDs.generate(TimeUUIDs.java:49)
	at org.apache.cassandra.stress.generate.values.TimeUUIDs.generate(TimeUUIDs.java:29)
	at org.apache.cassandra.stress.generate.PartitionIterator$MultiRowIterator.fill(PartitionIterator.java:653)
 at org.apache.cassandra.stress.generate.PartitionIterator$MultiRowIterator.fill(PartitionIterator.java:622)
	at org.apache.cassandra.stress.generate.PartitionIterator$MultiRowIterator.reset(PartitionIterator.java:270)
 at org.apache.cassandra.stress.generate.PartitionIterator.reset(PartitionIterator.java:107)
	at org.apache.cassandra.stress.operations.PartitionOperation.reset(PartitionOperation.java:115)
 at org.apache.cassandra.stress.operations.PartitionOperation.ready(PartitionOperation.java:101)
	at org.apache.cassandra.stress.StressAction$StreamOfOperations.nextOp(StressAction.java:370)
	at org.apache.cassandra.stress.StressAction$Consumer.run(StressAction.java:483)
 Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.AssertionError: java.lang.reflect.InaccessibleObjectException: Unable to make field private int java.io.FileDescriptor.fd accessible: module java.base does not "opens java.io" to unnamed module @7b139eab [in thread "Thread-45"]
	at org.apache.cassandra.utils.FBUtilities.getProtectedField(FBUtilities.java:664)
 at org.apache.cassandra.utils.NativeLibrary.<clinit>(NativeLibrary.java:81)
	at org.apache.cassandra.utils.UUIDGen.hash(UUIDGen.java:388)
 at org.apache.cassandra.utils.UUIDGen.makeNode(UUIDGen.java:367)
	at org.apache.cassandra.utils.UUIDGen.makeClockSeqAndNode(UUIDGen.java:300)
 at org.apache.cassandra.utils.UUIDGen.<clinit>(UUIDGen.java:41)
	... 10 more
 java.lang.AssertionError: java.lang.reflect.InaccessibleObjectException: Unable to make field private int java.io.FileDescriptor.fd accessible: module java.base does not "opens java.io" to unnamed module @7b139eab
	at org.apache.cassandra.utils.FBUtilities.getProtectedField(FBUtilities.java:664)
 at org.apache.cassandra.utils.NativeLibrary.<clinit>(NativeLibrary.java:81)
 at org.apache.cassandra.utils.UUIDGen.hash(UUIDGen.java:388)
	at org.apache.cassandra.utils.UUIDGen.makeNode(UUIDGen.java:367)
	at org.apache.cassandra.utils.UUIDGen.makeClockSeqAndNode(UUIDGen.java:300)
: at org.apache.cassandra.utils.UUIDGen.<clinit>(UUIDGen.java:41)
	at org.apache.cassandra.stress.generate.values.TimeUUIDs.generate(TimeUUIDs.java:49)
	at org.apache.cassandra.stress.generate.values.TimeUUIDs.generate(TimeUUIDs.java:29)
: at org.apache.cassandra.stress.generate.PartitionIterator$MultiRowIterator.fill(PartitionIterator.java:653)
	at org.apache.cassandra.stress.generate.PartitionIterator$MultiRowIterator.fill(PartitionIterator.java:622)
: at org.apache.cassandra.stress.generate.PartitionIterator$MultiRowIterator.reset(PartitionIterator.java:270)
	at org.apache.cassandra.stress.generate.PartitionIterator.reset(PartitionIterator.java:107)
	at org.apache.cassandra.stress.operations.PartitionOperation.reset(PartitionOperation.java:115)
: at org.apache.cassandra.stress.operations.PartitionOperation.ready(PartitionOperation.java:101)
	at org.apache.cassandra.stress.StressAction$StreamOfOperations.nextOp(StressAction.java:370)
: at org.apache.cassandra.stress.StressAction$Consumer.run(StressAction.java:483)
: Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make field private int java.io.FileDescriptor.fd accessible: module java.base does not "opens java.io" to unnamed module @7b139eab
	at java.base/java.lang.reflect.AccessibleObject.throwInaccessibleObjectException(Unknown Source)
: at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(Unknown Source)
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(Unknown Source)
	at java.base/java.lang.reflect.Field.checkCanSetAccessible(Unknown Source)
: at java.base/java.lang.reflect.Field.setAccessible(Unknown Source)
	at org.apache.cassandra.utils.FBUtilities.getProtectedField(FBUtilities.java:659)
	... 15 more
: java.lang.NoClassDefFoundError: Could not initialize class org.apache.cassandra.utils.UUIDGen
: at org.apache.cassandra.stress.generate.values.TimeUUIDs.generate(TimeUUIDs.java:49)
: at org.apache.cassandra.stress.generate.values.TimeUUIDs.generate(TimeUUIDs.java:29)
 
 Results:
 at org.apache.cassandra.stress.generate.PartitionIterator$MultiRowIterator.fill(PartitionIterator.java:653)
	at org.apache.cassandra.stress.generate.PartitionIterator$MultiRowIterator.fill(PartitionIterator.java:622)
 at org.apache.cassandra.stress.generate.PartitionIterator$MultiRowIterator.reset(PartitionIterator.java:270)
	at org.apache.cassandra.stress.generate.PartitionIterator.reset(PartitionIterator.java:107)
	at org.apache.cassandra.stress.operations.PartitionOperation.reset(PartitionOperation.java:115)
 at org.apache.cassandra.stress.operations.PartitionOperation.ready(PartitionOperation.java:101)
	at org.apache.cassandra.stress.StressAction$StreamOfOperations.nextOp(StressAction.java:370)
	at org.apache.cassandra.stress.StressAction$Consumer.run(StressAction.java:483)
 Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.AssertionError: java.lang.reflect.InaccessibleObjectException: Unable to make field private int java.io.FileDescriptor.fd accessible: module java.base does not "opens java.io" to unnamed module @7b139eab [in thread "Thread-45"]
	at org.apache.cassandra.utils.FBUtilities.getProtectedField(FBUtilities.java:664)
 at org.apache.cassandra.utils.NativeLibrary.<clinit>(NativeLibrary.java:81)
	at org.apache.cassandra.utils.UUIDGen.hash(UUIDGen.java:388)
	at org.apache.cassandra.utils.UUIDGen.makeNode(UUIDGen.java:367)
 at org.apache.cassandra.utils.UUIDGen.makeClockSeqAndNode(UUIDGen.java:300)
	at org.apache.cassandra.utils.UUIDGen.<clinit>(UUIDGen.java:41)
 ... 10 more
 Exception in thread "Thread-28"
 java.lang.NoClassDefFoundError: Could not initialize class org.apache.cassandra.utils.UUIDGen
 at org.apache.cassandra.stress.generate.values.TimeUUIDs.generate(TimeUUIDs.java:49)
	at org.apache.cassandra.stress.generate.values.TimeUUIDs.generate(TimeUUIDs.java:29)
 at org.apache.cassandra.stress.generate.PartitionIterator$MultiRowIterator.fill(PartitionIterator.java:653)
	at org.apache.cassandra.stress.generate.PartitionIterator$MultiRowIterator.fill(PartitionIterator.java:622)
	at org.apache.cassandra.stress.generate.PartitionIterator$MultiRowIterator.reset(PartitionIterator.java:270)
 at org.apache.cassandra.stress.generate.PartitionIterator.reset(PartitionIterator.java:107)
	at org.apache.cassandra.stress.operations.PartitionOperation.reset(PartitionOperation.java:115)
 at org.apache.cassandra.stress.operations.PartitionOperation.ready(PartitionOperation.java:101)
	at org.apache.cassandra.stress.StressAction$StreamOfOperations.nextOp(StressAction.java:370)
	at org.apache.cassandra.stress.StressAction$Consumer.run(StressAction.java:483)
 Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.AssertionError: java.lang.reflect.InaccessibleObjectException: Unable to make field private int java.io.FileDescriptor.fd accessible: module java.base does not "opens java.io" to unnamed module @7b139eab [in thread "Thread-45"]
	at org.apache.cassandra.utils.FBUtilities.getProtectedField(FBUtilities.java:664)
	at org.apache.cassandra.utils.NativeLibrary.<clinit>(NativeLibrary.java:81)
 at org.apache.cassandra.utils.UUIDGen.hash(UUIDGen.java:388)
	at org.apache.cassandra.utils.UUIDGen.makeNode(UUIDGen.java:367)
	at org.apache.cassandra.utils.UUIDGen.makeClockSeqAndNode(UUIDGen.java:300)
 at org.apache.cassandra.utils.UUIDGen.<clinit>(UUIDGen.java:41)
	... 10 more
 Exception in thread "Thread-46"
 java.lang.NoClassDefFoundError: Could not initialize class org.apache.cassandra.utils.UUIDGen
 at org.apache.cassandra.stress.generate.values.TimeUUIDs.generate(TimeUUIDs.java:49)
	at org.apache.cassandra.stress.generate.values.TimeUUIDs.generate(TimeUUIDs.java:29)
```

see:
https://github.com/scylladb/scylla-dtest/pull/5854

see `test_bulk_round_trip_blogposts` in
https://jenkins.scylladb.com/job/scylla-staging/job/dtest-pytest-gating/913/artifact/logs-full.release.001/dtest-gw13.log

--- a/conf/jvm-clients.options
+++ b/conf/jvm-clients.options
@@ -11,6 +11,11 @@
 -Djdk.attach.allowAttachSelf=true
 --add-exports java.base/jdk.internal.misc=ALL-UNNAMED
 --add-opens java.base/jdk.internal.module=ALL-UNNAMED
+--add-opens=java.base/java.lang=ALL-UNNAMED
+--add-opens=java.base/java.util=ALL-UNNAMED
+--add-opens=java.base/java.io=ALL-UNNAMED
+--add-opens java.base/java.net=ALL-UNNAMED
+--add-opens java.base/sun.nio.ch=ALL-UNNAMED
 --add-exports java.base/jdk.internal.ref=ALL-UNNAMED
 --add-exports java.base/sun.nio.ch=ALL-UNNAMED
 --add-exports java.management.rmi/com.sun.jmx.remote.internal.rmi=ALL-UNNAMED

